terra-node-ci/README.md

```diff
@@ -0,0 +1,36 @@
+# terra-node-ci
+
+This dockerfile provides a docker image for terra projects to extend to use in CI it will copy code as well as run npm install by default.
+
+## API (of a sort)
+
+* Extends terra-node-base
+* On build: Copies . to /opt/module
+* On build: Runs Npm Install
+
+## Usage
+
+This docker image is intended for CI and as such is fairly traditional. This docker image is expected to be defined as the base image for a project specific Dockerfile; see example below.
+
+```dockerfile
+# use node as base image
+FROM cerner/terra-node-ci:1
+
+```
+
+Then, when the above Dockerfile is used to build the project-specific docker container, the resulting docker container is based on a node docker container, it copies the contents of the project and runs npm install to setup the project for testing.
+
+To run tests against this container simply run the npm task you expect to run, for example using docker compose:
+
+```yml
+version: '3'
+
+services:
+  test-ci:
+    build: ./
+    command: 'npm run wdio-default'
+```
+
+```bash
+docker-compose run -e FORM_FACTOR=tiny test-ci
+```
```

Comment:
I think we'll be the only ones using these images. Would it be worth documenting the versioning strategy they'll follow or will it follow a conventional docker tagging or semantic versioning strategy? It might not be worth the effort, there just seems to be a few too many versioning strategies in the docker ecosystem.

Reply:
We will use semver. number of pi is just crazy....
https://medium.com/@mccode/using-semantic-versioning-for-docker-image-tags-dfde8be06699